### PR TITLE
Abort installation if .NET 4.5 isn't installed.

### LINKED ISSUE #12256: Windows installer .NET 4.5 detection doesn't seem to work
I've seen more than one report of crashes on XP and other windows with only .NET 4.0.  The installer should be preventing these people from installing the new version in the first place.

### PR DESCRIPTION
**Pull request by pchote**
Closes #12256.

### CODE REVIEW

File: packaging/windows/OpenRA.nsi [hunk 1]
```diff
@@ -152,15 +152,8 @@ Section "-DotNet" DotNet
...
-		MessageBox MB_YESNO ".NET Framework v4.5 or later is required to run OpenRA. $\n \
-		Do you wish for the installer to launch your web browser in order to download and install it?" \
-		IDYES download IDNO error2
-	download:
-		ExecShell "open" "http://www.microsoft.com/en-us/download/details.aspx?id=30653"
-		Goto done
-	error2:
-		MessageBox MB_OK "Installation will continue, but be aware that OpenRA will not run unless .NET v4.5 \
-		or later is installed."
+		MessageBox MB_OK ".NET Framework v4.5 or later is required to run OpenRA."
```
Comment (Phrohdoh) [inline packaging/windows/OpenRA.nsi]: It would be helpful to include the now-removed link to the download in this text.

Comment (pchote) [inline packaging/windows/OpenRA.nsi]: I don't think we can put inline links in message boxes, and none of the http://nsis.sourceforge.net/Reference/MessageBox options are really appropriate.

Comment (pchote) [inline packaging/windows/OpenRA.nsi]: While I agree with you in principle, given that we don't have any good options my preference is strongly towards letting users search for themselves.  This only affects a subset of windows 7 users: Windows 8 and above ship with 4.5 or better.

Showing a dialog box with a non-clickable link is just almost as bad for usability as our current broken opt-out dialog.

**Comment by obrakmann** [comment]
[New installer for testing](https://ci.appveyor.com/api/buildjobs/s973exrltbyqt5bj/artifacts/OpenRA-1b579b8.exe)

**Review by obrakmann — APPROVED** [review]
Looks good to me, 👍 . Can't test it myself, unfortunately.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9830554dd9da3b118acabdd46f2f7d56e7a96ec4...e02859cda194c30cd877e24e47d0e22807ba1f5b)

**Pull request merged**